apis/Google.Cloud.Firestore/Google.Cloud.Firestore.IntegrationTests/Google.Cloud.Firestore.IntegrationTests.csproj

```diff
@@ -7,8 +7,9 @@
     <NoWarn>1701;1702;1705;xUnit2004;xUnit2013</NoWarn>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Google.Api.Gax.Grpc.Testing" Version="[4.0.0, 5.0.0)" />
-    <PackageReference Include="Google.Api.Gax.Testing" Version="[4.0.0, 5.0.0)" />
+    <PackageReference Include="Google.Api.Gax.Grpc.Testing" Version="[3.7.0, 4.0.0)" />
```

Comment:
These are out of date - please pull the upstream main branch and rebase on top of it, removing unnecessary changes.